nodejs/week3/homework-template/src/backend/api/reservations.js

```diff
@@ -0,0 +1,50 @@
+const express = require("express");
```

Comment:
Similar to the previous file consider input validation and responding with adequate error codes.